package.json

```diff
@@ -1211,6 +1211,7 @@
     "dependencies": {
         "@azure/arm-appinsights": "^5.0.0-alpha.20230530.1",
         "@azure/arm-appservice": "^15.0.0",
+        "@azure/arm-authorization-profile-2020-09-01-hybrid": "^2.1.0",
```

Comment:
Curious: Were you forced to use `2020-09-01-hybrid` for some reason?

Reply:
https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/authorization/arm-authorization
https://learn.microsoft.com/en-us/javascript/api/overview/azure/arm-authorization-readme?view=azure-node-latest

The non-hybrid version appears to be more recent to me, but I'm not really sure why both of these packages exist separately / what the difference is

Reply:
https://learn.microsoft.com/en-us/javascript/api/overview/azure/arm-authorization-readme?view=azure-node-latest
Yeah, not really sure what the difference is